docs/cli/edgedb_server/edgedb_server_reset_password.rst

```diff
@@ -1,47 +0,0 @@
-.. _ref_cli_edgedb_server_reset_password:
```

Comment:
This should be kept (although renamed to `instance reset-auth`)

Reply:
Yeah, I just wanted to confirm the new name. I'll add it back in a moment.

Reply:
I've added it back as `instance reset-auth`